src/components/Layout.tsx

```diff
@@ -338,15 +336,14 @@ const Layout: React.FunctionComponent<Props> = (props: Props): JSX.Element => {
           style={{ display: 'none' }}
         ></div>
 
-        {
-          !isInIframe && (
-            <Nav
-              customer={customer}
-              theme={theme}
-              router={router}
-            />
-          )
-        }
+        {/*{*/}
```

Comment:
isInIframe helps to know whether the app is running inside an iframe from another host.

It's commented out because WIP and Layout is from SSR version and not used in SSG (see PageLayout instead)